packages/contentstack-audit/src/util/flags.ts

```diff
@@ -1,8 +1,15 @@
-import { ux } from '@contentstack/cli-utilities';
+import { FlagDefinition, Flags, ux } from '@contentstack/cli-utilities';
 
 import { IFlags, IncludeFlags } from '../types';
 import { tableColumnDescriptions } from '../messages';
 
+type JSONFlagOptions = {
+  hidden?: boolean;
+  description?: string;
+};
```

Comment:
@antonyagustine, Can the JSONflagOptions type be moved to the types folder?
```suggestion
type JSONFlagOptions = {
  hidden?: boolean;
  description?: string;
};
```